app/controllers/bacons_controller.rb

```diff
@@ -11,26 +12,35 @@ def did_launch
     # Store the number of runs per action
     now = Time.now.to_date
     launches.each do |action, count|
-      entry = Bacon.where(action_name: action, launch_date: now).take
-      entry ||= Bacon.create!(action_name: action, launch_date: now, launches: 0, number_errors: 0)
-
-      entry.launches += count
+      entry = Bacon.find_or_create_by(action_name: action, launch_date: now, tool_version: tool_version(action))
```

Comment:
Did I understand correct that we could have the following in our database:

action, date, version, launches
`deliver`, `today`, `1.0`, 2
`deliver`, `today`, `1.1`, 4

So we have 2 rows if the version number is not the same, right?